server.js

```diff
@@ -97,14 +97,15 @@ server.register({ register: require('./api') }, function(err) {
     return;
   }
 
-  var endpoint = process.env.PUBLIC_PROJECT_ENDPOINT;
-  var port = endpoint.match(/:(\d+)/);
+  const endpoint = process.env.PUBLIC_PROJECT_ENDPOINT;
+  let port = endpoint.match(/:(\d+)/);
+
   if (!port) {
     return;
   }
 
-  var port = parseInt(port[1]);
-  require('mox-server').runServer(port, function() {
-    console.log('running mox server on port', port);
+  port = parseInt(port[1], 10);
```

Comment:
I will override this rule in this repo's eslint config file since we're in a node env which does not need the radix.